src/main/java/frc/robot/turret/TurretSubsystem.java

```diff
@@ -34,13 +43,16 @@ public Rotation2d getVelocity() {
         return inputs.velocity;
     }
 
+    public void setPower(double power) {
+        turret.setPower(power);
+    }
 
     public void handleRotateToPoint(Translation2d targetPoint, Translation2d robotPosition) {
```

Comment:
Why is there an `handle` in the name of the function. This function purpose is to rotate to angle

Reply:
in that case can i leave it handle'something'

Reply:
?